utils/src/main/kotlin/OrtAuthenticator.kt

```diff
@@ -146,9 +147,11 @@ fun getNetrcAuthentication(contents: String, machine: String): PasswordAuthentic
     }
 
     credentialsPerMachine[machine]?.let {
+        OrtAuthenticator.logger().debug() { "Found .netrc entry for $machine" }
```

Comment:
Please use `OrtAuthenticator.log.debug { ... }` and a dot at the end of log message sentences.